app/js/controllers/foyer/ressources/ressources.js

```diff
@@ -93,9 +93,7 @@ angular.module('ddsApp').controller('FoyerRessourcesCtrl', function($scope, $sta
         };
     });
 
-    $scope.montantInvalide = function(montant) {
```

Comment:
@MattiSG, 
I had trouble transmitting this function to this insulated scope of the directive, so as it was not doing much, I just got rid of the function.